compiler/rustc_parse/src/parser/path.rs

```diff
@@ -341,9 +342,13 @@ impl<'a> Parser<'a> {
         let snapshot = if is_first_invocation { Some(self.clone()) } else { None };
 
         debug!("parse_generic_args_with_leading_angle_bracket_recovery: (snapshotting)");
-        match self.parse_angle_args() {
+        match self.parse_angle_args(ident_sp) {
             Ok(args) => Ok(args),
             Err(ref mut e) if is_first_invocation && self.unmatched_angle_bracket_count > 0 => {
+                debug!(
+                    "parse_angle_args_with_recovery: Err on first_invocation and unmatched: {:?}",
+                    self.unmatched_angle_bracket_count
+                );
```

Comment:
Could you remove the newly added `debug!` statements?
The probably of them being useful outside of your specific debugging session is vanishingly small.